schemas/json/service_plan.erb

```diff
@@ -3,5 +3,6 @@
   "id": "<%= @service_plan.id %>",
   "create_json_schema": <%= @service_plan.create_json_schema.to_json %>,
```

Comment:
Do we have a `create_json_schema` method on `@service_plan`?

Reply:
We don't - but I make an `OpenStruct` with a method for that in the service class. That way it can use either base or modified as the `create_json_schema` field.